index.html

```diff
@@ -23,12 +23,12 @@
                     <button id="startTime-btn">&#9658;</button>                                                         
                 </div>            
             </div>          
-            <div class="center-box">
+            <div class="center-box"> <!-- Comment -->
```

Comment:
Why does this `div` have a class but its siblings have `id`s?  I'd suggest this be changed to an `id` as well (don't forget to update the CSS).`